sunpy/visualization/mapcubeanimator.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, mapcube, annotate=True, **kwargs):
         if annotate:
             self._annotate_plot(0)
 
-    def updatefig(self, val, im, slider):
+    def updatefig(self, val, im, *args,**kwargs):
```

Comment:
by that I mean this method can be overriden in superclasses which can do something with the slider argument.